arangod/Aql/IResearchViewOptimizerRules.cpp

```diff
@@ -602,11 +631,166 @@ bool noDocumentMaterialization(
   return modified;
 }
 
-}  // namespace
+enum class SearchFuncType { kInvalid, kScorer, kOffsetInfo };
```

Comment:
```suggestion
enum class SearchFuncType { kInvalid, kScorer, kOffsetInfo, };
```
If you put comma, it will be formatted to separate lines for values